--- ag.orig	2018-03-14 10:58:42.360695200 -0700
+++ ag	2018-03-14 10:59:43.194225400 -0700
@@ -117,12 +117,12 @@ mssql_start() {
 		$OCF_SUCCESS)
 			# Allow this replica to become a master
 			#
-			crm_master -v 10 -N "$OCF_RESKEY_CRM_meta_on_node"
+			crm_master -v 10 -l reboot
 			;;
 		$OCF_RUNNING_MASTER)
 			# This is already a primary. Set its master score higher than other replicas.
 			#
-			crm_master -v 20 -N "$OCF_RESKEY_CRM_meta_on_node"
+			crm_master -v 20 -l reboot
 			;;
 	esac
 
@@ -224,16 +224,16 @@ mssql_monitor() {
 		$OCF_SUCCESS)
 			# Allow this replica to become a master.
 			#
-			crm_master -v 10 -N "$OCF_RESKEY_CRM_meta_on_node"
+			crm_master -v 10 -l reboot
 			;;
 		$OCF_RUNNING_MASTER)
 			# This is a primary. Set its master score higher than other replicas.
 			#
-			crm_master -v 20 -N "$OCF_RESKEY_CRM_meta_on_node"
+			crm_master -v 20 -l reboot
 			;;
 		*)
 			# This replica is not healthy enough to be promoted to a master.
-			crm_master -v 0 -N "$OCF_RESKEY_CRM_meta_on_node"
+			crm_master -v 0 -l reboot
 			;;
 	esac
 
